### PR TITLE
feat: add custom labels based on request and response objects

### PR DESCRIPTION
**Pull request by orinciog**
Hello!

This PR resolves https://github.com/joao-fontenele/express-prometheus-middleware/issues/21 feature request.

Thank you.

### CODE REVIEW

**Comment by mman** [comment]
I have not yet started working on this but this looks like very similar how I would do it. I will leave the review on @joao-fontenele.

From where I stand,  please fix the indentation of the code to match the original, otherwise nice work !!!

File: README.md [hunk 1]
```diff
@@ -62,6 +63,11 @@ app.use(promMid({
    * E.g.: `app_prefix_http_requests_total`
    */
   // prefix: 'app_prefix_',
+  /**
+   * Can add custom labels with customLabels and transformLabels options
+   */
+  //customLabels:['contentType']
```
Comment (joao-fontenele) [inline README.md:69]: please add an extra space after `//`

```
// customLabels:['contentType']
// transformLabels:(labels, req, res)=>{labels['contentType']=req.headers['content-type']}
```

**Comment by orinciog** [comment]
@joao-fontenele 
Hi, Thank you for your suggestions. Have you looked over my second commit?

**Comment by joao-fontenele** [comment]
already merged on #29

**Pull request closed without merging**